### DISCUSSION TITLE
I can't log into Vercel. It just keeps redirecting me

### DISCUSSION THREAD

**Discussion by twigs67** [vercel/community#4896] [Account Issue]
### Question

I've tried different browsers.. clearing my cache. Nothing works. I verify my email and then it just redirects me to login. Any idea how I might fix this?

**Comment by jwhardwick** [comment]
having the same issue.

**Comment by amyegan** [comment]
We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
